bbridge_client.py

```diff
@@ -48,15 +48,19 @@ def __init__(self, token, host_url="http://bbridgeapi.cloudapp.net/v1"):
         self.__sentiment_analysis_url = "{}/nlp/sentiment".format(host_url)
         self.__name_entity_recognition_url = "{}/nlp/ner".format(host_url)
 
-    def response(self, request_id, inner_type=None):
+    def response(self, request_id, return_type=None):
         """
         :type request_id: str
-        :type inner_type: type | None
+        :type return_type: type | None
```

Comment:
Can we specify `type` to be `BBridgeEntity` and call `from_json` method directly?